### PR TITLE
CORE-7116 : UTXO Ledger Contract Verification

### PR DESCRIPTION
**Pull request by lankydan**
Reopened https://github.com/corda/corda-api/pull/690 because GitHub wouldn't show new commits.

### CODE REVIEW

File: ledger/ledger-utxo/src/main/kotlin/net/corda/v5/ledger/utxo/UtxoLedgerService.kt [hunk 1]
```diff
@@ -78,12 +78,14 @@ interface UtxoLedgerService {
     fun filterSignedTransaction(signedTransaction: UtxoSignedTransaction): UtxoFilteredTransactionBuilder
 
     /**
-     * Collects signatures, records and broadcasts to involved peers a [UtxoSignedTransaction].
+     * Collects verifies a transaction, signatures, records and broadcasts it to involved peers a [UtxoSignedTransaction].
```
Comment (blsemo) [inline ledger/ledger-utxo/src/main/kotlin/net/corda/v5/ledger/utxo/UtxoLedgerService.kt]: That sentence doesn't quite parse.

Comment (vlajos) [inline ledger/ledger-utxo/src/main/kotlin/net/corda/v5/ledger/utxo/UtxoLedgerService.kt]:
```suggestion
     * Verifies a transaction, collects signatures, records and broadcasts it to involved peers a [UtxoSignedTransaction].
```
?

**Review by blsemo — COMMENTED** [review]
Looks fine apart from one mangled kDoc

File: ledger/ledger-utxo/src/main/kotlin/net/corda/v5/ledger/utxo/UtxoLedgerService.kt (hunk 1 above)
Comment (lankydan) [reply ledger/ledger-utxo/src/main/kotlin/net/corda/v5/ledger/utxo/UtxoLedgerService.kt]: 😮‍💨

Comment (lankydan) [reply ledger/ledger-utxo/src/main/kotlin/net/corda/v5/ledger/utxo/UtxoLedgerService.kt]: Sorted/

**Review by blsemo — DISMISSED** [review]

**Review by vlajos — APPROVED** [review]

**Review by mnesbit — APPROVED** [review]
OK

**Pull request merged**